src/lib/cache/lru_k_cache.hpp

```diff
@@ -81,22 +108,28 @@ class LRUKCache : public AbstractCacheImpl<Key, Value> {
     }
 
     // Insert new item in cache.
-    entry_t entry{key, value, {_access_counter}};
-    handle_t handle = _queue.push(entry);
+    LRUKCacheEntry entry{key, value, {_access_counter}};
+    Handle handle = _queue.push(entry);
     _map[key] = handle;
   }
 
   Value& get(const Key& key) {
     ++_access_counter;
 
     auto it = _map.find(key);
-    handle_t handle = it->second;
-    entry_t& entry = (*handle);
+    Handle handle = it->second;
+    LRUKCacheEntry& entry = (*handle);
     entry.add_history_entry(_access_counter);
     _queue.update(handle);
     return entry.value;
   }
 
+  static const KeyValuePair get_value(const std::pair<Key, Handle>& p) {
```

Comment:
only used internally; imprecise naming; could be inlined into caller?

Reply:
Argh. Sorry, totally missed that one. Will move to dereference() as done in GD(F)S caches.